management/schemas/advertiser.yaml

```diff
@@ -58,6 +58,22 @@ schemas:
       IsDeleted:
         type: boolean
         default: false
+      CatalogAccess:
+        type: array
+        nullable: true
+        items:
+          type: object
+          properties:
+            CatalogId:
+              type: integer
+              format: int32
+              nullable: true
+            ProductQuery:
+              type: array
+              items:
+                type: array
+                items: string
```

Comment:
Is it ok to have no `[min|max]Items`? Same bellow.

Reply:
I guess on both levels, `ProductQuery` array and the string `items` inside that array.